code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -909,7 +909,7 @@ var/list/robot_verbs_default = list(
 
 /mob/living/silicon/robot/attack_animal(mob/living/simple_animal/M as mob)
 	if(M.melee_damage_upper == 0)
-		M.emote("[M.friendly] [src]")
+		M.emote("me", 1, "[M.friendly] [src]")

```

Comment:
ahhhhhhhhh
aaaaaahhhhhhhhhhhhh
god no why
there has to be a better way to do this than fucking `emote("me", 1, "`